conference.js

```diff
@@ -123,15 +123,15 @@ function muteLocalVideo (muted) {
  * @param {boolean} [requestFeedback=false] if user feedback should be requested
  */
 function hangup (requestFeedback = false) {
-    let promise = Promise.resolve();
-
-    if (requestFeedback) {
-        promise = APP.UI.requestFeedback();
-    }
-
-    promise.then(function () {
+    APP.conference._room.leave().then(() => {
         connection.disconnect();
+        let promise = Promise.resolve();
```

Comment:
This could be written in a simpler, probably more readable way: 

```
if (requestFeedback) {
  return APP.UI.requestFeedback();
} else {
   return Promise.resolve();
}
```

:)